src/Components/Activities/ActivitiesList.js

```diff
@@ -10,11 +10,11 @@ const ActivitiesList = () => {
     const [items, setItems] = useState([]);
 
     useEffect(async() => {
-        const { data }  =  await getAllActivities();
-        setItems(data)
+        getAllActivities()
+            .then(allActivities => setItems(allActivities));
     }, []);
 
-    const showItemsListComponent = (items) => (<ActivitiesCards activities={items} />);
+    const showItemsListComponent = (items) => (<ActivitiesCards activities={items || []} />);
```

Comment:
items always is going to be [] or items because its initialized as [] and its is only being set if the get is success. If you need any extra validation you should do on the service

Reply:
I added a validation to activityService and used it in ActivitiesCards.js instead of ActivitiesList.js